### PR TITLE
Add prompts and caching for multiple runs

### PR DESCRIPTION
**Pull request by donnie-msft**
Felt like adding a simple UI so can run multiple queries at once. Planning to add more queries for myself so figured should PR the prompt now :)

nkolev92, are you ok with adding this?

image

### CODE REVIEW

File: GithubIssueTagger/Program.cs [hunk 1]
```diff
@@ -16,16 +20,90 @@ static async Task Main(string[] args)
...
 
-            var client = new GitHubClient(new ProductHeaderValue("nuget-github-issue-tagger"))
+            _client = new GitHubClient(new ProductHeaderValue("nuget-github-issue-tagger"))
             {
                 Credentials = new Credentials(args[0])
             };
 
-            var unprocessedIssues = await IssueUtilities.GetUnprocessedIssues(client, "nuget", "home");
-            foreach(var issue in unprocessedIssues)
+            await PromptForQuery();
```
Comment (nkolev92) [inline GithubIssueTagger/Program.cs:28]: I don't mind the overall change, but I wouldn't do it by default. 
The default should ideally just complete and not block.

**Review by nkolev92 — APPROVED** [review]
One suggestion about the default.

File: GithubIssueTagger/Program.cs (hunk 1 above)
Comment (donnie-msft) [reply GithubIssueTagger/Program.cs:28]: I'm confused... there is no default. You have to enter a choice. I can add a default....Which one should be the default? AllUnprocessed?

Comment (nkolev92) [reply GithubIssueTagger/Program.cs:28]: Doesn't it block waiting for input?

Comment (donnie-msft) [reply GithubIssueTagger/Program.cs]: See if that's what you wanted...

Comment (nkolev92) [reply GithubIssueTagger/Program.cs]: yeah, that was the idea.

**Pull request merged**